src/compiler/types.ts

```diff
@@ -2137,6 +2137,7 @@ namespace ts {
         /* @internal */ exportSymbol?: Symbol;  // Exported symbol associated with this symbol
         /* @internal */ constEnumOnlyModule?: boolean; // True if module contains only const enums or other modules with only const enums
         /* @internal */ isReferenced?: boolean; // True if the symbol is referenced elsewhere
+        /* @internal */ isDiscardable?: boolean; // True if a Javascript class property can be overwritten by a method

```

Comment:
done. I ended up with `isReplaceableByMethod` but tell if you want me to change it.